### DISCUSSION TITLE
ISA deviation

### DISCUSSION THREAD

**Discussion by andynuaa** [junzis/openap#35] [Q&A]
Dear @junzis,

Thanks for your great tool which facilitates me a lot concerning fuel consumption calculation. While I notice ISA condition is consistently assumed in the code and declared in your project relevant paper, is it possible to take into account **ISA deviation** when computing fuel flow? I think I may need to overwrite the pressure and temperature calculation (add ISA temperature offset), but not sure if the altitude correlation factor *ff_corr_alt* should also be modified in `fuel.py`?  Looking forward to your suggestions!

Cheers
Yifan

**Comment by junzis** [comment]
Perhaps you can convert your altitude to the ISA pressure altitude before fuel estimation? I think this would be the easiest approach that doesn't require you to change the OpenAP code.

**Reply by andynuaa** [reply]
Thank you! I will have a try
